recipes/xrftomo/meta.yaml

```diff
@@ -0,0 +1,62 @@
+
+{% set name = "XRFtomo" %}
+{% set version = "v1.0.1" %}
+
+package:
+  name: {{ name|lower }} 
+  version: {{ version }}
+
+source:
+  url: https://github.com/tomography/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 986c4ea85c7fe8bcef274b7085209b74e8a93acd444ad1f9b947fb3f1352cbbb
+
+build:
+  noarch: python
+  number: 0
+  skip: False
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python 
+    - pip
+
+  run:
+    - dxchange
+    - dxfile
+    - h5py
+    - matplotlib
+    - numpy
+    - pandas
+    - pyqt >=5
+    - pyqtgraph
+    - python >=3.7
```

Comment:
You should not add a runtime requirement in this way. Do you really require `python>=3.7`? If not, then use `noarch` (ignore the python=2.7 it's already EOL?) If so, then you don't remove the `noarch` and use skip to skip python versions less than 3.7.

Reply:
You will need to rerender the feedstock whenever you change `skip` or `noarch`.

Reply:
no, python >=3.7 is not absolutely necessary, but python>=2.x is. I am getting an error when using `noarch` and `skip: true [selector]` so it probably wouldn't like something like  `skip: true [py>2.x]` unless that is something that can be ignored as well. 

Aren't all the checks require to pass before conda-forge can accept my PR, if that's the case then how can the failing py27 or linux build be ignored? 

How do I rerender the feedstock? I thought that was only necessary until after the PR was approved.

Reply:
My mistake. You were right the first time. You should add requirements on the Python version in `requirements/run` and `requirements/host`. Please follow this example: https://github.com/conda-forge/conda-forge.github.io/issues/486#issuecomment-344946793.

These pulls are merged by humans; you can ask whether you should expect the import of pyQt5 to work correctly on the feedstock. If not, you can disable that test for the Linux build.

To rerender, [ask the conda-forge-admin](https://conda-forge.readthedocs.io/en/latest/webservice.html) or use conda-smithy.